File: EstablishmentService/EstablishmentService/ClientApp/src/app/models/user/user.model.ts
import { EstablishmentModel } from "../establishment/establishment.model";
import { ImageModel } from "../image/image.model";

export class UserModel {
  public id: number;
  public login: string;
  public firstName: string;
  public secondName: string;
  public role: string;
  public image: ImageModel;
  public establishments: EstablishmentModel[];

  constructor() {
    this.id = null;
    this.login = "";
    this.firstName = "";
    this.secondName = "";
    this.role = "";
    this.image = null;
    this.establishments = null;
  }
}

File: EstablishmentService/EstablishmentService/Models/User/UserModel.cs
using EstablishmentService.Models.File;

namespace EstablishmentService.Models.User
{
    public class UserModel
    {
        public int Id { get; set; }
        public string Login { get; set; }
        public string Role { get; set; }
        public string FirstName { get; set; }
        public string SecondName { get; set; }
        public ImageModel Image { get; set; }
    }
}
